### PR TITLE
[Parish Pages] Livestream Button Update

### LINKED ISSUE #695: Update Livestream Button on Parish Pages
Hey, time to update this one—it's been too long. Any chance we can do the following:

* [x] Monday-Sunday 12:01 am : Add to Calendar button appears (stylized as it was before) and gives you the chance to add a reminder for the Sunday livestream to your calendar for Sunday at 10:00am ET with the link to themeetinghouse.com/live
* [x] Sunday 12:02am : that button turns into a Watch Live button (styled the same as calendar) which opens /live
* [ ] Just for Easter: can we also find a way to add the Good Friday date + time in the calendar (it's unique for some parishes)
* [x] Just for Good Friday: can the Watch Live button link directly to the parish Good Friday link?

Please let me know if any of these are possible. If none are, then can we at least update the Livestream button and stylize it properly.

### PR DESCRIPTION
**Pull request by itbel**
- Parish page `Add to calendar` button will change to 'Watch Live` and link to /live page on Sundays.
- Parish page 'Watch Live' link will point to parish-specific live stream links on Good Friday.

Links can be changed by editing each site location .json file.
Either add or edit a "customLiveLink" field at the same level as addToCalendar
e.g. `public/static/content/hamilton-mountain.json`

Capture

resolves #695

### CODE REVIEW

File: src/components/RenderRouter/HeroItem.tsx [hunk 1]
```diff
@@ -247,6 +247,7 @@ class HeroItem extends React.Component<Props, State> {
       this.downArrowScroll();
     };
     if (this.state.content.style === 'full') {
+      console.log(moment().weekday());
```
Comment (JonEsparaz) [inline src/components/RenderRouter/HeroItem.tsx:250]: I assume this was for testing/debugging. Perhaps remove?

File: src/components/RenderRouter/HeroItem.tsx [hunk 2]
```diff
@@ -277,19 +278,38 @@ class HeroItem extends React.Component<Props, State> {
...
-                    className="heroBlackBoxA inverted"
-                    to={this.state.content.link1Action}
-                  >
-                    {this.state.content.link1Text}
-                  </Link>
-                </div>
-              ) : null}
-              {this.state.content.addToCalendar ? (
+              {moment().weekday() === 0 && this.state.locationData.length ? ( // Is Sunday
+                <button
```
Comment (JonEsparaz) [inline src/components/RenderRouter/HeroItem.tsx:282]: For the purpose of accessibility (and semantic HTML) this shouldn't be a button.

Re-styled `<LinkButton>` perhaps? Or just a React Router `<Link>`.

File: src/components/RenderRouter/HeroItem.tsx [hunk 3]
```diff
@@ -277,19 +278,38 @@ class HeroItem extends React.Component<Props, State> {
...
+                  <img
+                    className="calendarImage"
+                    src="/static/svg/Play.svg"
+                    alt="Contact Icon"
+                  />
+                  Watch Live
+                </button>
+              ) : moment().format('YYYY-MM-DD') === '2021-04-02' &&
+                this.state.locationData.length ? ( // Is good friday
+                <button
```
Comment (JonEsparaz) [inline src/components/RenderRouter/HeroItem.tsx:296]: Again `<LinkButton>` or `<Link>` is more appropriate.

File: src/components/RenderRouter/HeroItem.tsx (hunk 1 above)
Comment (itbel) [reply src/components/RenderRouter/HeroItem.tsx:250]: Whoops 😅

File: src/components/RenderRouter/HeroItem.tsx (hunk 3 above)
Comment (itbel) [reply src/components/RenderRouter/HeroItem.tsx]: Thanks, Jon. I have made changes based on your suggestions. See 3e2040a937992cb02f1750c548651b2c11623bbb.
Let me know anything seems off.

Comment (JonEsparaz) [reply src/components/RenderRouter/HeroItem.tsx]: LGTM

**Comment by lucastbelem** [comment]
@CarrissaReffell

**Pull request merged**